api/client/search.go

```diff
@@ -27,10 +27,9 @@ func (r ByStars) Less(i, j int) bool { return r[i].StarCount < r[j].StarCount }
 // Usage: docker search [OPTIONS] TERM
 func (cli *DockerCli) CmdSearch(args ...string) error {
 	cmd := Cli.Subcmd("search", []string{"TERM"}, Cli.DockerCommands["search"].Description, true)
-	noTrunc := cmd.Bool([]string{"#notrunc", "-no-trunc"}, false, "Don't truncate output")
```

Comment:
`-notrunc` was deprecated and I removed it
`--no-trunc` wasn't in docker search and I kept it in https://github.com/docker/docker/pull/17724/files#diff-9207f9a977bc11e99cfb49522dd26292R30

why do we want to delete even `--no-trunc` if it wasn't listed?